src/libraries/icubmod/embObjMotionControl/eomcParser.cpp

```diff
@@ -1246,8 +1310,6 @@ bool Parser::getCorrectPidForEachJoint(PidInfo *ppids/*, PidInfo *vpids*/, TrqPi
         //eomc_ctrl_out_type_vel = 2,
         //eomc_ctrl_out_type_cur = 3
 
-    return true;
-
```

Comment:
We noticed there was this `return` before any of the error messages were printed, was this intentional or a small bug, @randaz81 ?

Reply:
Because of this, there was no hope to see error messages popping out.